CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ### Unreleased
 
+- Body encoding now repects requested encoding and defaults to UTF-8 ([#18](https://github.com/clj-commons/clj-http-lite/issues/18)) ([@lread](https://github.com/lread))
```

Comment:
typo

Reply:
I reworded, please lemme know if there is still something amiss.